test/functional/helpers/simulator.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+import { getDevices, shutdown } from 'node-simctl';
+import { resetXCTestProcesses } from '../../../lib/utils';
+import { shutdownSimulator } from '../../../lib/simulator-management';
+import { killAllSimulators as simKill } from 'appium-ios-simulator';
+
+
+async function killAllSimulators () {
```

Comment:
why should we duplicate this in tests? I'd rather move the common code to utils.